tests/10-EdvClient.spec.js

```diff
@@ -6,9 +6,10 @@ import mock from './mock.js';
 import {isRecipient} from './test-utils.js';
 
 describe('EdvClient', () => {
-  let invocationSigner, keyResolver = null;
+  let invocationSigner, keyResolver, kAK = null;
```

Comment:
I see this is a carry over, but style prohibits initializing multiple variables on the same line.

Reply:
Fixed 437f761